### PR TITLE
Fix XML code snippets

### PR DESCRIPTION
**Pull request by Youssef1313**
Contributes to #9744

### CODE REVIEW

File: docs/framework/windows-workflow-foundation/how-to-enable-persistence-for-workflows-and-workflow-services.md [hunk 1]
```diff
@@ -52,10 +52,10 @@ Persistence can be enabled using an application configuration file by adding the
     <behaviors>
       <serviceBehaviors>
         <behavior name="myBehavior">
-          <SqlWorkflowInstanceStore connectionString="Data Source=myDatabaseServer;Initial Catalog=myPersistenceDatabase">
+          <sqlWorkflowInstanceStore connectionString="Data Source=myDatabaseServer;Initial Catalog=myPersistenceDatabase" />
```
Comment (Youssef1313) [inline docs/framework/windows-workflow-foundation/how-to-enable-persistence-for-workflows-and-workflow-services.md]: I made the `s` lowercase as it's in https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/windows-workflow-foundation/sqlworkflowinstancestore

File: docs/framework/windows-workflow-foundation/how-to-enable-sql-persistence-for-workflows-and-workflow-services.md [hunk 2]
```diff
@@ -3,7 +3,6 @@ title: "How to: Enable SQL Persistence for Workflows and Workflow Services"
 ms.date: "03/30/2017"
 dev_langs:
   - "csharp"
-  - "vb"
```
Comment (Youssef1313) [inline docs/framework/windows-workflow-foundation/how-to-enable-sql-persistence-for-workflows-and-workflow-services.md]: Article doesn't have any vb code. So removed this.

File: docs/standard/serialization/xml-serialization-with-xml-web-services.md [hunk 3]
```diff
@@ -226,21 +226,22 @@ public class Order {
...
+
+  <s:schema targetNamespace="http://tempuri.org/encodedTypes">
+    <s:complexType name="SoapBookService">
+      <s:sequence>
+        <s:element minOccurs="1" maxOccurs="1" name="EncodedOrderID" type="s:string" />
+      </s:sequence>
+    </s:complexType>
+  </s:schema>
+</s:complexType>
+```
```
Comment (Youssef1313) [inline docs/standard/serialization/xml-serialization-with-xml-web-services.md]: Not 100% sure of the changes made here.

File: docs/framework/windows-workflow-foundation/how-to-enable-persistence-for-workflows-and-workflow-services.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/windows-workflow-foundation/how-to-enable-persistence-for-workflows-and-workflow-services.md]: My understanding is that they're probably case insensitive (I'd have to check) but change looks good.

File: docs/framework/windows-workflow-foundation/how-to-enable-sql-persistence-for-workflows-and-workflow-services.md (hunk 2 above)
Comment (mairaw) [reply docs/framework/windows-workflow-foundation/how-to-enable-sql-persistence-for-workflows-and-workflow-services.md]: you can remove the whole thing then. csharp is the default one: https://github.com/dotnet/docs/blob/master/docfx.json#L97

File: docs/standard/serialization/xml-serialization-with-xml-web-services.md (hunk 3 above)
Comment (mairaw) [reply docs/standard/serialization/xml-serialization-with-xml-web-services.md]: definetely looks better than before.

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @Youssef1313. Left one suggestion for you.

**Review by mairaw — APPROVED** [review]

**Pull request merged**